helm/hpcc/templates/datastorage-pvc.yaml

```diff
@@ -3,3 +3,5 @@
 {{- if (and (not $dataStorage.existingClaim) (not $dataStorage.plane)) }}
 {{ include "hpcc.addPVC" (dict "root" $ "name" "datastorage" "me" $dataStorage) }}
 {{- end }}
+{{- include "hpcc.addPVCsFromPlanes" (dict "root" $ "label" "data") }}
+{{- include "hpcc.addPVCsFromPlanes" (dict "root" $ "label" "lz") }}
```

Comment:
rather than have a hpcc.addPVCsFromPlanes in each data/dll/dalistorage (feels like these files should be retired) for some specific labels (e.g. "data" and "lz" here in datastorage-pvc.yaml),
would it be better to have a storage.yaml, that creates PVC for all planes defined regardless of labels, as long as they define storageClass ?

Reply:
I think that can/should be a separate pr.